ReSharper.FSharp/src/FSharp.Common/src/FcsReactorMonitor.fs

```diff
@@ -161,3 +156,48 @@ type FcsReactorMonitor
                 override __.Dispose () = match operations.TryRemove operationId with _ -> ()
                 override __.OperationName = sprintf "{%d}%s" operationId opName
             }
+
+        override __.OnOperationStart opName opArg = onOperationStart opName opArg
+        override __.OnOperationEnd() = onOperationEnd()
+
+
+[<ShellComponent>]
+type FcsReactorListener(logger: ILogger, reactorMonitor: IFcsReactorMonitor) =
```

Comment:
Given both the listener and the monitor are shell components now, could it be a single component instead?